currency/pairs.go

```diff
@@ -159,6 +170,19 @@ func (p Pairs) GetPairsByFilter(filter Code) Pairs {
 	return pairs
 }
 
+// GetPairsByCurrencies returns all pairs that have both matches to the
+// currencies passed in. This allows for the construction of enabled pairs by
+// required currency codes.
+func (p Pairs) GetPairsByCurrencies(enabled Currencies) Pairs {
```

Comment:
`enabled` here bears no meaning, passing in a `Currencies` does not mean that they are enabled. Please rn to a more descriptive variable such as `currencies`